### DISCUSSION TITLE
Crypto donations

### DISCUSSION THREAD

**Discussion by TheFuzzStone** [FossifyOrg/General-Discussion#41] [General]
Yeah, I saw that there are a couple ways to support Fossify.

I suggest adding some crypto as donations, namely:

- Monero (XMR) // GrapheneOS has a positive experience, see here: https://twitter.com/GrapheneOS/status/1684663777273233408
- Bitcoin (BTC)
- Ethereum (ETH) // on which you will be able to accept ERC20 tokens, namely the most liquid ones are: USDT, USDC and DAI
- Tron (TRX) // which has the most liquid and widely used USDT on its network

**Comment by inson1** [comment]
I woul like - Monero, bitcoin, etherium

**Comment by ghost** [comment]
I will fully admit I'm not in the crypto space like at all, but from what I know, pretty much all Crypto Currency is basically a scam. Maybe not some of the currencies themselves but in my limited looking I haven't really found a way to get into it that doesn't involve a major investment of existing capital that could vanish the next day. Also, I've been starting to see reports that Cryptocurrencies are highly destructive to the environment which is already dying.

On top of those, Crypto has always been volatile, I'd be worried both as a donor and the receiver about the value, I wouldn't want to give a donation equating to 100 USD and it ends up being 2 USD by the time the receiver gets it.

If any of my assessments are incorrect I apologize, these are my passive observations from the outside, Feel free to correct me.

**Reply by inson1** [reply]
@TheFuzzStone 
But we need financial system. And it has to be online. And goverments and central banks arent your friends. Better ideas?

> major investment of existing capital that could vanish the next day

every invesment ever

> Also, I've been starting to see reports that Cryptocurrencies are highly destructive to the environment which is already dying.

If you dont have central power, you need something else. They just need computing power - electricity and graphic cards/CPUs. Bitcoin uses 0,5% of world electricity, that should be 0,1% world power usage. Better then people dead, because of bad goverment that can control everything imo. Etherium way of solving it is just bad imo.
Internet, cars etc. also needs power. Are they bad?

> On top of those, Crypto has always been volatile, I'd be worried both as a donor and the receiver about the value

This part is on donor and if naveensingh wants to accept it.

**Reply by ghost** [reply]
> But we need financial system. And it has to be online. And goverments and central banks arent your friends. Better ideas?

I wish I had a better idea. The Entire Concept of Money and Currency is kinda tainted now.

>every invesment ever

Most investments don't evaporate within hours. Though, I would like to point out, you consider currency an investment if I read that correctly. However I will concede on this point.

>If you dont have central power, you need something else. They just need computing power - electricity and graphic cards/CPUs. Bitcoin uses 0,5% of world electricity, that should be 0,1% world power usage. Better then people dead, because of bad goverment that can control everything imo. Etherium way of solving it is just bad imo.

I don't know the math here, but I do know Crypto has in fact caused decommissioned coal power plants to come back, solely to make more crypto and not help anyone else. Would it have anyway? who knows, but it was recommissioned for Cytpo and that kind of behavior will kill people.

> Internet, cars etc. also needs power. Are they bad?

Actually I do believe Cars are generally bad, or at least not good. They have some uses, however I believe alternate forms of transport are far far far better, both for Freight and People. We don't need as many Cars as we have.
As for the Internet, that largely depends on what part you're talking about in my opinion.
I would also like to point out, it's not exactly the power usage that's the problem, but the power generation. Most of the world's power still comes from fossil fuels., and as stated above, Crypto has reactivated fossil fuel plants. If the power was 100% renewables like Wind and Solar the power draw wouldn't be a problem, but that's not the world we live in.


Ultimately I am against Cypto, and I doubt anyone can change my mind on in a Github Discussion. However I also recognize it is not my decision. If the Project wants to use Crypto as a method of donation, it doesn't change much. However it's worth considering the opposing view point, especially those that might screw over unsuspecting donors, the project, or even the earth at large (You can look up the state of climate change yourself if your interested).

**Reply by inson1** [reply]
@hayleethegamer 

> I wish I had a better idea. The Entire Concept of Money and Currency is kinda tainted now.

We need money. So yes please find better solution with same privacy, decentralized and with less consumption or idk if you dont have alternatives

> Most investments don't evaporate within hours. Though, I would like to point out, you consider currency an investment if I read that correctly. However I will concede on this point.

Btw still more trustworthy then some goverments. 
They can. And what did you see was just death of few projects. Projects are dying every day. And that is normal. Do you remember SMT? xD

Currency in its infancy. (that is the reason for volatilty) That is deflationary (if you count max supply). And if you count last 10 years, it would be the best investment.

> I don't know the math here

Search it then.

>  I do know Crypto has in fact caused decommissioned coal power plants to come back, solely to make more crypto and not help anyone else. Would it have anyway? who knows, but it was recommissioned for Cytpo and that kind of behavior will kill people.

Yes, there was maybe few reactivated coal power plants. For example Miners/Miner on Island are using geotermal power. In the end they are just going to use the cheapest electricity. Why no just tax more non clean electricity?

Hey if you would rather have North Korea kind of situation then 0.1% increase in energy consumption then have it :)

> As for the Internet, that largely depends on what part you're talking about in my opinion.

You are player right?

If people would rather work than were playing games and watching anime/serials/films then there wouldnt be global warming. /s
What about Tor? What about Fossify and SMT? How much power we could save? And naveensingh could just work on clean solutions from the donations /s - Privacy and decentralization matters

I will rather help clean some part of city/park etc. from trash, recycle, try not buy so much things and plastic things. Try buying more second hand things. Use public transport. etc.

On other hand, if there isnt monero (or github doesnt 2x my donations) I wouldnt donate. In the end - **less options -> less donations**

And with that last part I would end this disscussion :) it doesnt add to much to this project right?

**Reply by ghost** [reply]
> And with that last part I would end this disscussion :) it doesnt add to much to this project right?

Agreed. We have a difference of opinion and we've both made our points, it's up to those that make decisions to take in the information and decide.

**Reply by inson1** [reply]
@hayleethegamer I hope you also agree with last point. more options -> more donations

**Reply by TheFuzzStone** [reply]
@hayleethegamer 

>Ultimately I am against crypto

That's the beauty of Monero (XMR) (I'm not interested in other cryptos) is that they don't care what anyone else thinks about them, and that you can't shut them down. :)

I am purposely not commenting on the world agenda of "green communism". No desire to waste my time on that. 

>more options -> more donations

That's the only reason I created this discussion. Let everyone choose for themselves how to support the project. But financial support is needed, since as far as I understand it was the main reason for the sale of SimpleMobile.

**Reply by inson1** [reply]
@TheFuzzStone 
> That's the only reason I created this discussion. Let everyone choose for themselves how to support the project.

Yes, that disscussion was useless, I should start and end with that point.

> But financial support is needed, since as far as I understand it was the main reason for the sale of SimpleMobile.

naveensingh said that he doesnt thinks so https://github.com/orgs/FossifyOrg/discussions/14#discussioncomment-7768295
But yes money are needed

> communism

You commented on it.

**Reply by ghost** [reply]
> I am purposely not commenting on the world agenda of "green communism". No desire to waste my time on that.

All I'm going to say is you may be mistaken in whatever lead you to that idea. There's stark differences in Socialism, Communism, Capitalism, and just hearing really really bad things and being concerned about implied support. 

Also did I misunderstand something about SMT's sale, I was under the impression it was less about needing money to continue more burn out/wanting to stop but wanting to be compensated for hard work.

**Reply by inson1** [reply]
@hayleethegamer 

> but wanting to be complicated for hard work.

what?

**Reply by ghost** [reply]
@inson1 
Sorry, I must of Typoed "Compensated" and my phone Autocorrected it without my noticing, I have fixed this error.

**Reply by inson1** [reply]
@TheFuzzStone ok I get it now

**Comment by naveensingh** [comment]
@TheFuzzStone I'll add crypto options as well. Preferences aside, which ones are the most popular?

**Reply by inson1** [reply]
@naveensingh preferences aside, it does depends how many do you want to add, but I think most worth it are Bitcoin, Etherium and Monero

> https://twitter.com/GrapheneOS/status/1684663777273233408

TheFuzzStone shared good example for that

**Reply by naveensingh** [reply]
I think 2-3 would be enough but we can also add the fourth one.

**Reply by inson1** [reply]
@naveensingh In the end I think creating/maintaining wallets isnt that hard, so if someone wants to do big donation I would just create the wallet

The worst case scenario is death of the crypto currency = same as if the person didnt donate anything = So lets be user friendly :)

So adding somewhere some rule about min ammount for new crypto would be good - 50 usd/eur ?

PS To avoid someone using it as ad for his crypo I would only added (for example) 100 biggest by maret cap and sorted it by market cap - if there would be interest

**Reply by naveensingh** [reply]
@inson1 I'll create a separate Donation.md file in the main project repo with a lot of donation links. In apps and everywhere else, the most popular (not preferred) options will be shown with a **More options** link at the bottom that directs to the donation list in the repo. What do you think?

**Reply by TheFuzzStone** [reply]
> @TheFuzzStone I'll add crypto options as well. Preferences aside, which ones are the most popular?

* Monero (XMR)
* Ethereum (ETH) 
    * creating ETH address you automatically can accept ERC20 tokens like the most popular stablecoins: USDT, USDC and DAI
* Tron (TRX)
    * you want to create this TRX address because you will be able to accept donations of the same USDT as in the Ethereum network, but the fee for the one who will make this donation will be many times lower
* Bitcoin (BTC)

----

Thousands of people can come here and shill their favorite crypto projects here, and that the project will "definitely get donations", but it will all be more the wish of individuals rather than reality. 
Also, over time, you can add other wallets, it's very easy to do, the main thing is that there are people willing to donate.

Once you (@naveensingh) make a Monero (XMR) wallet, ping me please, I'll make the first donation. :)

**Reply by bardesss** [reply]
Make sure you add Bitcoin Lightning. Onchain payments are not for small payments.

**Reply by inson1** [reply]
@naveensingh We dont know what are the popular options, right? We dont have good enough data. Do you think donations would be lower if everything would be in the donation file? Plus (future) Most asked questions in repository readme wouldnt be visible (without scrolling) on notebooks. How many options would be in Readme?

PS It should be DONATION.md so its same as README.md, CONTRABUTING.md or CHANGELOG.md etc.

**Reply by inson1** [reply]
@naveensingh ping

**Reply by naveensingh** [reply]
> Do you think donations would be lower if everything would be in the donation file?

Yes. Not everything would be in the donation file. 

> How many options would be in Readme?

Current options plus some crypto options.

The bottom line is, we'll see what works. I'll set up crypto options soon, if anyone has any advice on how to set things up, please let me know as I'm not a crypto ninja (I mostly invest in the stock market and occasionally do trading for the fun of it).

**Comment by TheFuzzStone** [comment]
> advice on how to set things up

For Ethereum, Tron, Bitcoin - use multi-currency FOSS wallet - https://unstoppable.money/
For Monero use: https://www.monerujo.io/ or https://monero.com/

If you have some questions, you can DM me directly: https://thefuzzstone.github.io/

**Comment by naveensingh** [comment]
@TheFuzzStone @inson1 I have added the recommended options to the readme (addresses only): https://github.com/FossifyOrg#crypto-options

I plan to organize these options later using a separate donation page. I'll now close this issue. Feel free to create a new issue if you want more options.

**Reply by TheFuzzStone** [reply]
@naveensingh, thanks for not letting the project die.

---

https://xmrchain.net/search?value=ec6910150056840abbb5931bd662b41bbb66f5e81187c0ce7396fb38bc7d90b4

🫡
